bolt.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "net-ssh", ">= 4.0"
   spec.add_dependency "orchestrator_client", "~> 0.3.1"
   spec.add_dependency "puppet", [">= 6.0.1", "< 7"]
+  spec.add_dependency "puppet-resource_api"
```

Comment:
Why? A mention in the commit would help.

Reply:
it's needed to use the panos module and is part of the puppet 6 package